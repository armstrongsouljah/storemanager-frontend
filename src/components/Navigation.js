/* eslint-disable jsx-a11y/anchor-is-valid */
/* eslint-disable react/jsx-indent */
import React from "react";

const Navigation = (props) => (
    <nav>
        <div className="nav-wrapper black white-text">
            <a href="#" className="brand-logo">Dashboard</a>
            <ul id="nav-mobile" className="right hide-on-med-and-down">
                <li><a href="#" onClick={props.onLogout} className="btn btn-small blue-grey white-text">Logout</a></li>
            </ul>
        </div>
    </nav>
);

export default Navigation;
