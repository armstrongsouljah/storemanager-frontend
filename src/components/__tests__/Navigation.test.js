import React from 'react';
import { shallow } from 'enzyme';

import Navigation from '../Navigation';

describe('<Navigation/>', () => {
    it("should render without crashing", () => {
        const wrapper = shallow(<Navigation/>);
        expect(wrapper).toMatchSnapshot();
    })
})
