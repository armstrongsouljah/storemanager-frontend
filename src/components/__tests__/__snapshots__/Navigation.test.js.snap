// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation/> should render without crashing 1`] = `ShallowWrapper {}`;
